modules/theme/src/themes/wso2is/assets/images/illustrations/ai-loading-screen-placeholder.svg

```diff
@@ -0,0 +1,111 @@
+    <path d="M180.838 169.656L185.838 164.656" stroke="#40404B" stroke-width="2" stroke-linecap="round"/>
+    <defs>
+    <linearGradient id="paint0_linear_2373_75" x1="96.8766" y1="119.382" x2="309.283" y2="291.746" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#EE575C"/>
+    <stop offset="1" stop-color="#F67147"/>
+    </linearGradient>
+    <linearGradient id="paint1_linear_2373_75" x1="254.473" y1="69.4799" x2="263.311" y2="103.831" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FF9D44"/>
+    <stop offset="1" stop-color="#F5C58D"/>
+    </linearGradient>
+    <linearGradient id="paint2_linear_2373_75" x1="345.802" y1="151.121" x2="345.802" y2="180.507" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FF9D44"/>
+    <stop offset="1" stop-color="#F5C58D"/>
+    </linearGradient>
+    <linearGradient id="paint3_linear_2373_75" x1="130.979" y1="56.9377" x2="130.979" y2="86.4065" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FF9D44"/>
+    <stop offset="1" stop-color="#F5C58D"/>
+    </linearGradient>
+    <linearGradient id="paint4_linear_2373_75" x1="105.613" y1="281.063" x2="112.647" y2="318.406" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FF9D44"/>
+    <stop offset="1" stop-color="#F5C58D"/>
+    </linearGradient>
+    <linearGradient id="paint5_linear_2373_75" x1="70.8379" y1="154.656" x2="70.8379" y2="186.656" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FF9D44"/>
+    <stop offset="1" stop-color="#F5C58D"/>
+    </linearGradient>
+    <linearGradient id="paint6_linear_2373_75" x1="55.3379" y1="180.656" x2="55.3379" y2="198.656" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FF9D44"/>
+    <stop offset="1" stop-color="#F5C58D"/>
+    </linearGradient>
+    <linearGradient id="paint7_linear_2373_75" x1="287.338" y1="305.656" x2="287.338" y2="344.656" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FF9D44"/>
+    <stop offset="1" stop-color="#F5C58D"/>
+    </linearGradient>
+    <linearGradient id="paint8_linear_2373_75" x1="207.838" y1="204.656" x2="207.838" y2="276.758" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#F67149"/>
+    <stop offset="1" stop-color="#F05D58"/>
+    </linearGradient>
+    <linearGradient id="paint9_linear_2373_75" x1="207.838" y1="204.656" x2="207.838" y2="276.758" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FCA959"/>
+    <stop offset="1" stop-color="#FFB669"/>
+    </linearGradient>
+    <linearGradient id="paint10_linear_2373_75" x1="207.838" y1="157.707" x2="207.838" y2="221.424" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#FCA959"/>
+    <stop offset="1" stop-color="#FFB669"/>
+    </linearGradient>
+    <linearGradient id="paint11_linear_2373_75" x1="208.838" y1="120.656" x2="208.838" y2="200.656" gradientUnits="userSpaceOnUse">
+    <stop stop-color="#F75857"/>
+    <stop offset="1" stop-color="#FF805B"/>
+    </linearGradient>
+    </defs>
+    </svg>
+    
```

Comment:
Seems the newline is not properly added here.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5961/commits/891f646b2c63094d4467060253aa82c4bd6751b5